lib/web/router/impl.ex

```diff
@@ -232,10 +243,11 @@ defmodule Antikythera.Router.Impl do
           controller :: v[module],
           action :: v[atom],
           path_matches :: Keyword.t(String.t()),
-          websocket? :: v[boolean]
+          websocket? :: v[boolean],
+          timeout :: v[GearActionTimeout.t()] \\ GearActionTimeout.default()
         ) :: route_result do
     if Enum.all?(path_matches, fn {_placeholder, match} -> String.printable?(match) end) do
-      {controller, action, Map.new(path_matches), websocket?}
+      {controller, action, Map.new(path_matches), websocket?, timeout}
```

Comment:
I think we should check `GearActionTimeout.max/0` at runtime as well.
For example, if a gear was compiled with 5000ms, then we change the setting in the antikythera instance to 3000ms. I think we want to change the timeout of the gear to 3000ms without recompiling the gear.

Reply:
Thank you for your review.
As you said, it is necessary to check if the timeout is in range also at runtime.
Fixed at e073767.